Helper/DeletedProductSerializer.php

```diff
@@ -26,7 +26,7 @@ public function serialize($deletedProductOrders)
                     
                     if ($parentProduct) {
                         $productOptions[] = [
-                            'id'       => $itemId,
+                            'id'       => ($itemSku) ? $itemSku : $itemId,
```

Comment:
Those are some magic parentheses?

Reply:
Parentheses are used only if there is expression, removing them.